apps/examples/testcase/le_tc/kernel/tc_libc_sched.c

```diff
@@ -108,6 +108,25 @@ static void tc_libc_sched_task_setcanceltype(void)
 
 	TC_SUCCESS_RESULT();
 }
+
+/**
+ * @fn                   :tc_libc_sched_task_testcancel
+ * @brief                :This tc tests tc_libc_sched_task_testcancel()
+ * @Scenario             :The task_testcancel() function creates a cancellation point in the
+ *                        calling thread. The task_testcancel() function has no effect if
+ *                        cancelability is disabled
+ * @API'scovered         :task_testcancel
+ * @Preconditions        :none
+ * @Postconditions       :none
+ * @return               :void
+ */
+
+static void tc_libc_sched_task_testcancel(void)
+{
+	task_testcancel();
```

Comment:
This was removed by https://github.com/Samsung/TizenRT/commit/3ffed1349c59b477213d9246c5982065be3b15bf#diff-aafb30cd3a9000323b9006de33be0fdeafa0e0489b84f161984a05605e1a0960. TC should include atleast one TC_ASSERT_XX macro. But this is not. Let's remove. How about removing function itself?

Reply:
yes if the function is not there , we can remove